pkg/util/printer/printer.go

```diff
@@ -84,8 +84,8 @@ func GetTiDBInfo() string {
 		config.GetGlobalConfig().Store,
 		enterpriseVersion,
 	)
-	if config.IsCloudStore() {
-		info += "\nCloud Store: true"
+	if config.IsObjectStore() {
+		info += "\nData Storage: Object Storage"
```

Comment:
Recently I am learning CSE code, I see they refer the underlying layer as DFS (seems distributed file system). Not sure if our teams have agreeement on the naming 😂